app/Repository/Eloquent/FolderRepository.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repository\Eloquent;
+
+use App\Services\EvidenceFileHandler;
+use App\Folder;
+
+class FolderRepository
+{
+    /**
+     * @var EvidenceFileHandler
+     */
+    private $evidenceFileHandler;
+
+    public function __construct(EvidenceFileHandler $evidenceFileHandler)
+    {
+        $this->evidenceFileHandler = $evidenceFileHandler;
+    }
```

Comment:
The property is not used and only set in the constructor. Better to remove both the property and constructor.